### ISSUE TITLE
Cannot login to 1.2.18

### ISSUE THREAD

**Issue by BryanGriffith** [chadsmith/OpenVBX-Android#12]
OpenVBX installations updated to version 1.2.18, and reports incorrect e-mail/password. I have another installation still running 1.2.14 and I can login to that server.

**Comment by obrun** [comment]
Similarly account information which works on the web and in iOS does not work in the Android version.

When the Android 6 permissions were modest changes in LoginActivity.java. There is now a private boolean flag canCheckPhoneState which handles situations where we do not have permission to check the phone status. It is initialized by default to zero. There is not anything obviously wrong that I can see.

End points have / appended and then index.php appended. This can result in index.php/ which would appear to be unintentional.

Then there are any changes in the openvbx api. Passwords at schema revision 63 became salted hashes of forty characters in length but the code attempts to be backward compatible.

In short I and several commenters in the App Store have a similar issue but I can not see the cause in the code. Logging in is unfortunately often the most complex thing that most IT environments ever have to do and fraught with unexpected testing scenarios.

The error message itself just means that any kind of exception was thrown during the execution of OpenVBX.API.getFolders() which appears to means what directories are below /messages/inbox but at that point it looses me. It seems like it is trying to read "folders" below /messages/inbox/.

Basically there appears to be a lot that can go wrong there that has nothing do with the login itself.

**Comment by xerojin** [comment]
Still not working. I'm on version 1.2.20

Any insight on this issue?
